### ISSUE TITLE
[Combat] Leash Range

### ISSUE THREAD

**Issue by rychersky** [Atlantiss/NetherwingBugtracker#4748]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: Some enemy mobs will continue to chase you far past their intended leash range. I have experienced this while using Piercing Howl to kite enemies and they have chased me well in excess of 100 yards from when I last damaged them and/or their spawn point.

**Server Revision**: 2745

**Comment by rychersky** [comment]
I was engaged by a Hound of Culuthas from the Ruins of Farahlon as well as an addition mob (the eye things). I didn't really want to fight them so I used intimidating shout and ran back. The Hound followed me for a good 300 yards before I opted to just kill it. I did not do any further damage to the mob, just running and occasionally using Piercing Howl.

Pictured below are the mob chasing me well beyond its spawn point (Ruins of Farahlon are far back in the background). Additionally, I have included a map picture to show the distance traveled while kiting.

![wowscrnshot_020119_192914](https://user-images.githubusercontent.com/45608488/52159517-bbbeb980-2662-11e9-8f32-268dad891e47.jpg)
![wowscrnshot_020119_192917](https://user-images.githubusercontent.com/45608488/52159518-bbbeb980-2662-11e9-9b61-20534f2dfea1.jpg)
